### DISCUSSION TITLE
Where is the tool width pulled from?

### DISCUSSION THREAD

**Discussion by crcarmichael** [Courseplay/courseplay#7484] [Q&A] [answered]
Hello!  I have a sprayer mod that has an actual working width of 18m.  CP course generator detects a width of 8.9m. I've changed some references in the mod XML file from 9 to 18 (the store now shows the correct width) but CP is still detecting 8.9m.  Where do I have to change this value in the mod so that CP gets the correct width? TIA!

**Comment by Tensuko** [comment, marked as answer]
Did you press the orange button on the course generator screen ? It should detect the correct width.

**Reply by crcarmichael** [reply]
Yes. It's the detected width that is wrong, so when I press the orange button it sets at 8.9m instead of 18m. So I just need to know where to look in the mod files to fix the width so that CP can detect the correct width. I changed it already in the mod XML... so it must be in the i3d?

**Reply by Tensuko** [reply]
I think it was the AI Markers...

But before you chage stuff in the mod, there is a configfile.
It have to be in ...\My Games\FarmingSimulator2019\modsSettings\Courseplay

If not, here is the blank version:
[VehicleConfigurations.txt](https://github.com/Courseplay/courseplay/files/7167137/VehicleConfigurations.txt)
NOTE: You have to change it back from .txt to .XML (github don't like .XML).

For how it works, you can check the same file in the FS19_Courseplay.zp file ...\courseplay\config
